modules/internal_registry/internal-cr-token.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION=$(jq -r '.tsb_version')
+
+if [[ "${VERSION}" =~ .*"-dev" ]]; then
+    REGISTRY="gcr.io/tetrate-internal-containers"
+    TOKEN=$(gcloud auth print-access-token)
+fi
```

Comment:
Default to empty values to not require `gcloud` if it is not a dev version, and also to allow decoupling the registry from the version in other modules. With this other modules can just use `if an internal registry has been set, use it`, without having to parse the TSB version.